c7n/cli.py

```diff
@@ -326,6 +326,10 @@ def setup_parser():
     run.set_defaults(command="c7n.commands.run")
     _default_options(run)
     _dryrun_option(run)
+    run.add_argument(
+        "-V", "--skip-validation",
```

Comment:
i think we want to leave off on a short form here, ie. remove '-V', disabling validation should be pretty explicit.